ci-operator/config/openshift-cnv/cnv-ci/openshift-cnv-cnv-ci-master.yaml

```diff
@@ -15,6 +15,9 @@ tag_specification:
   namespace: ocp
 tests:
   - as: "e2e-test"
+    secret:
+      name: openshift-cnv-dev-cnv-bot-pull-secret
+      mount_path: /etc/cnv-installer/pull-secret
```

Comment:
According to https://github.com/openshift/ci-tools/blob/master/CONFIGURATION.md#testsecrets

Shouldnt it be

```
    secrets:
      - name: openshift-cnv-dev-cnv-bot-pull-secret
         path: /etc/cnv-installer/pull-secret
```

Reply:
you are correct. fixing it.